src/main/java/org/embulk/input/kintone/KintoneClient.java

```diff
@@ -113,4 +134,37 @@ public void deleteCursor(String cursor)
             this.logger.error(e.toString());
         }
     }
+
+    public Map<String, FieldProperty> getFields(final PluginTask task)
+    {
+        Map<String, FieldProperty> fields = this.appClient.getFormFields(task.getAppId());
+        if (task.getExpandSubtable()) {
+            Map<String, FieldProperty> subtableFields = new HashMap<>();
+            List<String> subtableFieldCodes = new ArrayList<>();
+            for (Map.Entry<String, FieldProperty> fieldEntry : fields.entrySet()) {
+                if (fieldEntry.getValue().getType() == FieldType.SUBTABLE) {
+                    subtableFields.putAll(((SubtableFieldProperty) fieldEntry.getValue()).getFields());
```

Comment:
Are field codes unique across the app, including within subtable?

Reply:
It appears so.  You cannot set the same filed code to a column of subtable as one of not-subtable.
But I couldn't find the official document...